docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+| `given`    | The “given” name. Can be multiple words such as “Mary Ann”.<br>Examples:  “Janus”, “Mary Jean”, or “Jean-Louis”|
+| `given2`   | Additional given name or names or middle name, usually names(s) written between the given and surname. Can be multiple words. In some references, also known as a “second” or “additional” given name or patronymic. This field is separate from the “given” field because it is often optional in various presentation forms.<br>Examples:  “Horatio Wallace” as in<br>`{ given: "Janus", given2: "Horatio Wallace", surname: "Young" }`<br><br>“S.” as in “Harry S. Truman”. Yes, his full middle name was legally just “S.”.|
+| `surname`  | The “family name”. Can be more than one word.<br><br>Example: “van Gogh” as in<br>`{ given: "Vincent", given2: "Willem", surname: "van Gogh" }`<br><br>Other examples: “Heathcote-Drummond-Willoughby” as in “William Emanuel Heathcote-Drummond-Willoughby III”|
+| `surname2` | Secondary surname (used in some cultures), such as second or maternal surname in Mexico and Spain. This field is separate from the “surname” field because it is often optional in various presentation forms, and is considered a separate distinct name in some cultures.<br><br>Example: “Barrientos” in “Diego Rivera Barrientos”;<br>`{ given: "Diego", surname: "Rivera", surname2: "Barrientos" }`<br><br>Example: if "Mary Jane Smith" moves to Spain the new name may be<br>`{ given: "Mary", given2: "Jane", surname: "Smith", surname2: "Jones" }`|
+| `suffix`   | Typically a title, honorific, or generational qualifier.<br>Example: “PhD”, “Jr.”<br><br>Example: “Sonny Jarvis Jr.”<br>`{ given: "Salvatore", given2: "Blinken", surname: "Jarvis", suffix: "Jr." }`<br><br>An alternate PersonName object may be presented for formatting using the “stage” name from the application’s data:<br>`{ given: "Salvatore", given-informal: "Sonny", given2: "", surname: "Jarvis", suffix: "Jr." }` |
+
+Some other examples:
+
+* British name: _John Ronald Reuel Tolkien_: `given` name is "John", `given2` name would be  "Ronald Reuel", and the `surame` is "Tolkien".
+* Dutch name: _Anneliese Louise van der Pol_: `given` name: "Anneliese", `given2` name: "Louise", `surname`: "van der Pol"
+    * Also surname-prefix: “van der”, surname-core: “Pol” — see below.	
+* French name: “Jean-Louis Trintignant” would _not _be Jean (`given`) Louis (`given2`) Trintignant (`surname`), since “Louis” wouldn’t be discarded when formatting. Instead it would be Jean-Louis (`given`) Trintignant (`surname`)
+
+Note: If the legal name, stage name, etc. are substantially different, then that information can be logically in a separate Name record. That is, it is up to the implementation to maintain any distinctions that are important to it: CLDR PersonName formats is focusing on formatting a Name record that is given to it. 
```

Comment:
Check for "name record" and change to "name object"

Reply:
Actually I changed to PersonName object everywhere (per your later comment), except in the first occurrence in the Intro where we are speaking generally, which I just changed to name object.